boards/arm/stm32l4/nucleo-l476rg/src/stm32_appinit.c

```diff
@@ -448,6 +452,21 @@ int board_app_initialize(uintptr_t arg)
     }
 #endif
 
+#ifdef CONFIG_SENSORS_MPU9250
+  /* Try to register MPU9250 device in I2C1 */
+
+  ret = board_mpu9250_initialize(0, 1);
+
```

Comment:
Please remove this empty line between return and the if test. I missed it in your bmp280_initialize, so please remove it there too.